src/tests/main.test.ts

```diff
@@ -101,6 +101,11 @@ describe("Purchases.isEntitledTo", () => {
 });
 
 describe("Purchases.changeUser", () => {
+  test("throws error if given invalid user id", async () => {
+    const purchases = configurePurchases();
+    await expect(purchases.changeUser("")).rejects.toThrowError(PurchasesError);
```

Comment:
Does this need the `await`?

Reply:
If I do not add it the exception is happening outside of the test context and the test fails given that `changeUser` is returning a promise.

Reply:
But I can definitely write this in a better way. Give me a min.

Reply:
Ahh right, nvm then, I think it's ok!

Reply:
Actually I could not make it better. But I have updated the happy path test to also await to ensure there are no exceptions happening there.